frontend/packages/kubevirt-plugin/integration-tests/tests/utils/constants/vm.ts

```diff
@@ -122,3 +122,11 @@ export const diskTabCol = {
   storageClass: 5,
 };
 Object.freeze(diskTabCol);
+
+export const wrongValues = ['Capitalletter', 'dasattheend-'];
+
+// export const wrongValues = [
```

Comment:
?

Reply:
This is the list of wrong values that are used for negative testing for names, K8S rules - no capital letters and no special symbols at the end